include/core/truthTable/truth_table.hpp

```diff
@@ -50,6 +52,18 @@ namespace syrec {
                 }
                 return cube;
             }
+
+            // construct a cube from a string
+            static auto fromString(const std::string& str) -> Cube {
+                assert(str.size() <= 64U);
```

Comment:
why the limitation to less than 64 bits? Strings in particular are the way to specify cubes with more than 64bits.

Reply:
done 👍🏼